src/xscen/biasadjust.py

```diff
@@ -5,14 +5,14 @@
 
 import xarray as xr
 import xclim as xc
-from xclim import sdba
+import xsdba
 
 from .catutils import parse_from_ds
 from .config import parse_config
 from .utils import minimum_calendar, standardize_periods
 
 logger = logging.getLogger(__name__)
-xc.set_options(sdba_encode_cf=True, sdba_extra_output=False)
+xsdba.set_options(xsdba_encode_cf=True, xsdba_extra_output=False)
```

Comment:
```suggestion
xsdba.set_options(xsdba_extra_output=False)
```
As a side note, I suggest we change this option name to remove "xsdba" ?